PhysicsTools/Heppy/python/analyzers/gen/LHEWeightAnalyzer.py

```diff
@@ -19,28 +19,49 @@ class LHEWeightAnalyzer( Analyzer ):
     def __init__(self, cfg_ana, cfg_comp, looperName ):
         super(LHEWeightAnalyzer,self).__init__(cfg_ana,cfg_comp,looperName)
 
+        self.LHEWeightsNames=[]
+        self.lumiCounter=-1
+        self.currentLumi=0
```

Comment:
What are `self.lumiCounter` and `self.currentLumi` used for?
If, as it seems, they are not used for producing any of the output of the module then they should be removed.